sdk/resourcemanager/Azure.ResourceManager/src/Azure.ResourceManager.csproj

```diff
@@ -22,6 +22,7 @@
     <Compile Include="$(AzureCoreSharedSources)ReferenceTypeAttribute.cs" LinkBase="Shared\Core" />
     <Compile Include="$(AzureCoreSharedSources)SerializationConstructorAttribute.cs" LinkBase="Shared\Core" />
     <Compile Include="$(AzureCoreSharedSources)TypeReferenceTypeAttribute.cs" LinkBase="Shared\Core" />
+    <Compile Include="$(AzureCoreSharedSources)ExperimentalAttribute.cs" LinkBase="Shared\Core" />
```

Comment:
given how much pain we are going through to remove shared source, is it a good idea to add more right now?

Reply:
This will be needed in Azure.Provisioning as well.